spec/reek/smells/nested_iterators_spec.rb

```diff
@@ -49,27 +49,9 @@ def chained
   it 'detects an iterator with an empty block' do
     src = <<-EOS
       def foo
-        bar { baz { } }
-      end
-    EOS
-    expect(src).to reek_of(:NestedIterators)
-  end
-
-  it 'should report nested iterators only once per method' do
-    src = <<-EOS
-      def bad(fred)
-        @fred.each {|item| item.each {|part| @joe.send} }
-        @jim.each {|ting| ting.each {|piece| @hal.send} }
-      end
-    EOS
-    expect(src).to reek_of(:NestedIterators)
-  end
-
-  it 'reports nested iterators only once per method even if levels are different' do
-    src = <<-EOS
-      def bad(fred)
-        @fred.each {|item| item.each {|part| part.foo} }
-        @jim.each {|ting| ting.each {|piece| piece.each {|atom| atom.foo } } }
```

Comment:
Those specs were useless in their former form. I moved them down and adjusted them.